app/src/main/java/com/cornellappdev/scoop/ui/screens/PostScreen.kt

```diff
@@ -53,33 +56,26 @@ fun PostScreen(
 ) {
     val pagerState = rememberPagerState(0)
     val coroutineScope = rememberCoroutineScope()
-    val rideState = remember {
-        mutableStateOf(
-            Ride()
-        )
-    }
 
     Box {
         Column {
-            HorizontalPager(
-                count = 3, state = pagerState,
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .weight(1f), userScrollEnabled = false
-            ) { page ->
+            AnimatedVisibility(visible = pagerState.currentPage == 0 || pagerState.currentPage == 1) {
                 Column(
                     modifier = Modifier
                         .fillMaxWidth()
                         .padding(top = 40.dp),
-                    verticalArrangement = Arrangement.Top,
-                    horizontalAlignment = Alignment.CenterHorizontally
+                    verticalArrangement = Arrangement.Bottom,
+                    horizontalAlignment = Alignment.End
                 ) {
-                    Text(
-                        text = stringResource(R.string.trip_details),
-                        style = TextStyle(color = Color.Black, fontSize = 24.sp),
-                        textAlign = TextAlign.Center
-                    )
+                    Image(painter = painterResource(R.drawable.header), contentDescription = "Header")
                 }
+            }
+            HorizontalPager(
+                count = 3, state = pagerState,
```

Comment:
I think the formatting is a little weird here. Did you turn on format file before pushing?